app/helpers/homepage_helper.rb

```diff
@@ -0,0 +1,23 @@
+module HomepageHelper
+  def organisations_path
+    Plek.find("content-store") + "/content/government/organisations"
+  end
+
+  def organisations_json
+    JSON.parse(RestClient.get(organisations_path))
+  end
+
+  def ministerial_departments_count
+    organisations_json
+      .dig("details", "ordered_ministerial_departments")
+      .filter { |org| org["govuk_status"] != "joining" }
```

Comment:
This is here because there seems to be a presenter in [collections](https://github.com/alphagov/collections/blob/53189e4b723d67d27a50cedb57981f39dface7f9/app/presenters/organisations/index_presenter.rb#L51-L55) that filters by this before we render on: https://www.gov.uk/government/organisations#agencies_and_other_public_bodies

🤔 I'm not sure why this would be the case though. Anyone know the history of this?

Reply:
Joining organisations aren't yet public on GOV.UK if I remember correctly, so I think this is why they're not shown in the list yet.

Reply:
ahh I see, thanks @thomasleese!